test/jdk/jdk/incubator/vector/benchmark/src/main/java/benchmark/bigdata/BooleanArrayCheck.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2020, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
The copyright should be "Copyright (c) 2021, " since this is a new file.

Reply:
done